### PR TITLE
Update EasyPQP step

### PR DESCRIPTION
**Pull request by Leon-Bichmann**
This PR is updating the EasyPQP step to generate libraries from DDA search results + some small fixes and changes.

## PR checklist
 - [x] This comment contains a description of changes (with reason)
 - [x] If you've fixed a bug or added code that should be tested, add tests!
 - [x] If necessary, also make a PR on the [nf-core/diaproteomics branch on the nf-core/test-datasets repo]( https://github.com/nf-core/test-datasets/pull/new/nf-core/diaproteomics)
 - [x] Ensure the test suite passes (`nextflow run . -profile test,docker`).
 - [x] Make sure your code lints (`nf-core lint .`).
 - [x] Documentation in `docs` is updated
 - [x] `CHANGELOG.md` is updated
 - [x] `README.md` is updated

**Learn more about contributing:** https://github.com/nf-core/diaproteomics/tree/master/.github/CONTRIBUTING.md

### CODE REVIEW

File: main.nf [hunk 1]
```diff
@@ -31,8 +31,9 @@ def helpMessage() {
       --irt_min_rsq			Minimal rsq error for irt RT alignment (default=0.95)
       --irt_alignment_method            Method for irt RT alignment ('linear','lowess')
       --generate_spectral_lib           Set flag if spectral lib should be generated from provided DDA data (pepXML and mzML)
-      --dda_pepxmls                     Path to DDA pepXML input for library generation
-      --dda_mzmls                       Path to DDA mzML input for library generation
+      --mzid                            Path to mzid to generate spectral library (mzid)
+      --dda_mzml                        Path to mzml to generate spectral library (mzml)
+      --unimod                          Path to unimod.xml file describing modifications
```
Comment (timosachsenberg) [inline main.nf]: maybe add where to download

File: nextflow.config [hunk 2]
```diff
@@ -31,11 +31,15 @@ params {
   pyprophet_peakgroup_fdr = 0.01
   pyprophet_peptide_fdr = 0.01
   pyprophet_protein_fdr = 0.01
+  pyprophet_pi0_start = 0.1
```
Comment (timosachsenberg) [inline nextflow.config:35]: can you add some comments / link where the defaults are from?

File: main.nf [hunk 3]
```diff
@@ -31,8 +31,9 @@ def helpMessage() {
       --irt_min_rsq			Minimal rsq error for irt RT alignment (default=0.95)
       --irt_alignment_method            Method for irt RT alignment ('linear','lowess')
       --generate_spectral_lib           Set flag if spectral lib should be generated from provided DDA data (pepXML and mzML)
-      --dda_pepxmls                     Path to DDA pepXML input for library generation
-      --dda_mzmls                       Path to DDA mzML input for library generation
+      --mzid                            Path to mzid to generate spectral library (mzid)
```
Comment (timosachsenberg) [inline main.nf]: can you support both idXML and mzid?

**Review by timosachsenberg — APPROVED** [review]
look good - minor comments

File: nextflow.config (hunk 2 above)
Comment (Leon-Bichmann) [reply nextflow.config:35]: I took the pyprophet parameter from here:
https://github.com/PyProphet/pyprophet/blob/fa86fcc5c9185083540ca5be3f1f7a740c0fba41/pyprophet/main.py#L56

File: main.nf (hunk 3 above)
Comment (Leon-Bichmann) [reply main.nf]: jup, will add a better description

File: main.nf (hunk 1 above)
Comment (Leon-Bichmann) [reply main.nf]: will add a description

**Pull request merged**